src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/DF_BaseRecalibrator.java

```diff
@@ -76,68 +84,134 @@
      */
     private String beforePath = null;
 
-
     /**
      * Path to the reference (SAM file). Eventually can be filesystem or gs://
      */
     private String referencePath = null;
 
     /**
-     * Path to save the output to. Must be local for now.
+     * Path to save the serialized tables to. Local or GCS.
      */
-    @Argument(doc = "Path to save the output to. Must be filesystem.", optional = false)
-    protected String outputTablesPath = "df-recaltables.sj";
+    @Argument(doc = "Path to save the serialized recalibrationTables to.", optional = true)
```

Comment:
Define explicit `fullName` and `shortName` for all arguments that current lack it.